jina/serve/runtimes/gateway/request_handling.py

```diff
@@ -157,18 +159,23 @@ async def _load_balance(self, request):
         try:
             async with aiohttp.ClientSession() as session:
 
-                if request.method == 'GET':
-                    request_kwargs = {}
-                    try:
-                        payload = await request.json()
-                        if payload:
-                            request_kwargs['json'] = payload
-                    except Exception:
-                        self.logger.debug('No JSON payload found in request')
-
-                    async with session.get(
-                        url=target_url, **request_kwargs
-                    ) as response:
+                request_kwargs = {
+                    'headers': request.headers,
+                    'params': request.query,
+                }
```

Comment:
The original code did not forward the headers & query, which makes me wonder if it's intentional?

Reply:
it was not intentional